artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/proton/AMQPTunneledCoreLargeMessageReader.java

```diff
@@ -98,9 +98,11 @@ private enum State {
    private LargeServerMessage coreLargeMessage;
    private int largeMessageSectionRemaining;
    private State state = State.CLOSED;
+   private final boolean largeMessageSync;
```

Comment:
You don't need to track this value, just call the accessor in the Session context